hack/generate/samples/ansible/constants.go

```diff
@@ -518,6 +518,25 @@ const rolesForBaseOperator = `
       - watch
 #+kubebuilder:scaffold:rules
 `
+const rolesForProject = `
```

Comment:
This is an upstream project, and we wouldn't put anything OCP specific here. If that is needed, it should go in the downstream fork.